packages/dd-trace/src/dogstatsd.js

```diff
@@ -190,6 +190,14 @@ class CustomMetrics {
   constructor (config) {
     const clientConfig = DogStatsDClient.generateClientConfig(config)
     this.dogstatsd = new DogStatsDClient(clientConfig)
+
+    this._boundFlush = this.flush.bind(this)
+
+    // TODO(bengl) this magic number should be configurable
```

Comment:
This must match the timing of the agent, so it could be a footgun to make it configurable.

Reply:
I guess the comment is not really blocking this from landing?